daml-assistant/src/DA/Daml/Assistant/Version.hs

```diff
@@ -181,3 +191,28 @@ getLatestSdkSnapshotVersion = do
     pure $ do
         versions <- eitherToMaybe versionsE
         maximumMay versions
+
+latestReleaseVersionUrl :: String
+latestReleaseVersionUrl = "https://docs.daml.com/latest"
+
+getLatestReleaseVersion :: IO SdkVersion
+getLatestReleaseVersion = do
```

Comment:
On second thought, there's some discrepancy here between using `getLatestReleaseVersion` and `getLatestSdkVersionCached` defined above. Maybe the `getLatestReleaseVersion` should just be the non-cached version of `getLatestSdkVersionCached`:

```
getLatestSdkVersion :: DamlPath -> IO (Maybe SdkVersion)
getLatestSdkVersion damlPath = do
    versionsE <- tryAssistant $ getAvailableSdkVersions damlPath
    pure $ do
        versions <- eitherToMaybe versionsE
        maximumMay versions
```

That way it would always line up with `daml version`, I think?